pkg/client/docs/models/operations/v2reverttransactionresponse.md

```
@@ -6,4 +6,5 @@
 | Field                                                                                             | Type                                                                                              | Required                                                                                          | Description                                                                                       |
 | ------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
 | `HTTPMeta`                                                                                        | [components.HTTPMetadata](../../models/components/httpmetadata.md)                                | :heavy_check_mark:                                                                                | N/A                                                                                               |
-| `V2CreateTransactionResponse`                                                                     | [*components.V2CreateTransactionResponse](../../models/components/v2createtransactionresponse.md) | :heavy_minus_sign:                                                                                | OK                                                                                                |
\ No newline at end of file
+| `V2CreateTransactionResponse`                                                                     | [*components.V2CreateTransactionResponse](../../models/components/v2createtransactionresponse.md) | :heavy_minus_sign:                                                                                | OK                                                                                                |
+| `Headers`                                                                                         | map[string][]*string*                                                                             | :heavy_check_mark:                                                                                | N/A                                                                                               |
```

markdownlint-cli2 (0.18.1)

Reference links and images should use a label that is defined
Missing link or image reference definition: "string"

(MD052, reference-links-images)